publiccode.yml

```diff
@@ -112,3 +121,12 @@ dependsOn:
   open:
    - name: Kubernetes
    - name: Helm
+
+metadataFiles:
```

Comment:
This block is already present in the current file

Reply:
Fixed